docs/hiring-guide/community-spotlight/interview-chris-bahnweg.rst

```diff
@@ -0,0 +1,86 @@
+    went back to the drawing board and considered getting another degree in software engineering to make 
+    myself more marketable as a tech writer. Thankfully it didn't have to come to that. That was my plan, 
+    and I'm trying to remember how I met Alex. I think he had pointed something out to me, and we got to 
+    talking and we became friends. He would update me on his job search, because he was also searching for 
+    a job at that point. I would ask about his job search and provide advice where I thought it was 
+    appropriate. And the story kind of came down the way I explained before.
+
+**Were you getting help from other members of the community?**
+
+    I was getting advice from a lot of people. I would say the career-advice channel is really helpful if 
+    you're trying to get help in software engineering or technical writing. But it's also kind of helpful 
+    to pick someone and establish a kind of camaraderie with them. I've met a bunch of people on here that 
+    I would consider my friends, and I think that is really helpful, because even though you might never 
+    meet these people in "real life", you never know what opportunities they may have. I've checked up on 
+    people and have asked if they know any opportunities at their company. On the flipside. I have a couple
+    friends on Write the Docs who are developers. If I see a developer job that fits what they're looking 
+    for, I'll say, "I saw this job posting. Take a look at it."
+
+**What pieces of advice were you getting?**
+
+    I'd say the biggest one is build a portfolio, because that's how people are going to judge you. I say 
+    this to my HR person all the time: I can say that I'm a level 10 Docker wizard that can orchestrate 
+    applications with ease, but unless I have code that demonstrates my knowledge of Docker, you're not 
+    going to believe me. Or I can say I know all the DITA in the world, but unless I have documentation 
+    that showcases my knowledge of DITA, or I have a certification that showcases my knowledge of DITA, 
+    you're not going to believe me. So the portfolio gives you an anchor point for your skills. It gives 
+    you a way to say that you did this and here's proof. I think that is a really good starting point. 
+    Beyond that, I would say definitely learn a programming language of some kind. Now, that doesn't mean 
+    become a full stack developer. But it does mean know some Javascript, know some Python, know what a 
+    REST API might look like. But also be knowledgeable about the programming scene in your area. If you're 
+    in Columbus, Ohio and the programming scene is full of Ruby on Rails developers, it doesn't behoove you 
+    to learn Javascript. It behooves you to learn Ruby on Rails, because that's probably what the APIs in 
+    the area are going to be built in.
+
+Advice for grads starting to build their careers
```

Comment:
I agree w/ Sam, this feels a bit weird as a heading without a question. Is there a question that you asked folks, perhaps that should be included, instead of a heading going directly into an answer.

Reply:
I'm open to removing the headings. I can also go through and add in the questions for context where necessary .

Reply:
@ericholscher I just meant either have Q&A, or have Headings and more summary. But not both. 

Overall, sounds like we're all leaning towards Q&A?

Reply:
Without the headings, the transitionary paragraphs are a little difficult to pick out. Do you have any ideas for making them stand out against the quotes? Should I just make them bold with the questions?

Reply:
I suspect we're in danger of overthinking it. I'm happy with this as is. Or you could make it Italic if you really want it to stand out. I don't even object to bold. :man_shrugging: